pkg/aws/actuator/actuator.go

```diff
@@ -190,8 +190,10 @@ func (a *AWSActuator) needsUpdate(ctx context.Context, cr *minterv1.CredentialsR
 			UserName: aws.String(awsStatus.User),
 		})
 		if err != nil {
-			logger.WithError(err).Errorf("error getting user: %s", user)
-			return true, fmt.Errorf("unable to read info for username %v: %v", user, err)
+			// Could not fetch current user, we unset the user in the status and create a new one
+			logger.WithError(err).Errorf("error getting user: %s, creating a new user", awsStatus.User)
```

Comment:
Can we do a more specific `if errror.IsNotFound()` type of check so that a transient AWS error doesn't cause CCO to create a new IAM User?